prompting/__init__.py

```diff
@@ -36,4 +36,29 @@
 from . import agent
 from . import conversation
 from . import dendrite
+
 from .llms import hf
+
+from .tasks import TASKS
+from .tools import DATASETS
+from .task_registry import TASK_REGISTRY
+
+# Assert that all tasks have a dataset, and all tasks/datasets are in the TASKS and DATASETS dictionaries.
+registry_missing_task = set(TASK_REGISTRY.keys()) - set(TASKS.keys())
+registry_extra_task = set(TASKS.keys()) - set(TASK_REGISTRY.keys())
+assert (
+    not registry_missing_task
+), f"Missing tasks in TASK_REGISTRY: {registry_missing_task}"
+assert not registry_extra_task, f"Extra tasks in TASK_REGISTRY: {registry_extra_task}"
+
+registry_datasets = set(
+    [dataset for task, datasets in TASK_REGISTRY.items() for dataset in datasets]
+)
+registry_missing_dataset = registry_datasets - set(DATASETS.keys())
+registry_extra_dataset = set(DATASETS.keys()) - registry_datasets
+assert (
```

Comment:
I'm not a fan of this. I know I wrote it. Can you verify that this is stable?

Reply:
It's stable, and does work as intended (That's how I found out we needed a mock task to take the mock dataset in the registry). I move a copy of it to the test though so we don't have to have this assertion in the init.